### PR TITLE
Add indented wrapping - server

### LINKED ISSUE #26: Add indented wrapping for long messages in server app
It should not head towards the left side where the time and username are displayed when wrapping. Instead, the wrap should be indented to push it forward by at least 22 characters.

### PR DESCRIPTION
**Pull request by melsaa**
## Description
Add indented wrapping for long messages in the server app.

Fixes #26 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
I have tested it locally without any error or warning.

**Test Configuration**:
* OS: Ubuntu
* OS version: 16.04
* Python version: 3.7.8
* Architecture: 64-bit

### CODE REVIEW

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Take a look at this.

image

There's an extra line which forms up even after the entire text has been completely displayed. Please try and remove that.

**Comment by melsaa** [comment]
Thanks for the feedback. I have removed the extra line.

File: main.py [hunk 1]
```diff
@@ -38,6 +38,24 @@ async def notify_mesej(message):
...
+
+    for i in range(0, message_width, width):
+        if i > 0:
+            wrapped_message += indent_text
+        wrapped_message += message[i : i + width]
+        if i < message_width - width:
+            wrapped_message += '\n'
+
+    return wrapped_message
+
```
Comment (t0xic0der) [inline main.py:58]: Works like a charm. 

image

**Review by t0xic0der — APPROVED** [review]
Thank you for the changes you made. I am approving your pull request.

File: main.py (hunk 1 above)
Comment (melsaa) [reply main.py:58]: Thank you!

**Pull request merged**